mm2src/adex_cli/Cargo.toml

```diff
@@ -22,7 +22,7 @@ sysinfo = "0.28"
 tiny-bip39 = "0.8.0"
 tokio = { version = "1.20", features = [ "macros" ] }
 
-[target.'cfg(all(not(target_arch = "wasm32"), not(windows)))'.dependencies]
+[target.'cfg(all(not(target_arch = "wasm32"), not(target_os = "macos"), not(windows)))'.dependencies]
```

Comment:
I think you should simply check for target that `fork` suppose to work on.

Reply:
@ozkanonur, it should be quite better now. Look please )